Dockerfile

```diff
@@ -32,8 +32,7 @@ RUN set -x \
  && /fossology/install/scripts/php-conf-fix.sh --overwrite \
  && /fossology/utils/fo-installdeps -e -y \
  && $_cleanup
-RUN curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer
+RUN make composer_install
```

Comment:
The script `fo-installdeps` already handles the installation of composer, i.e. writes it to `/usr/local/bin/composer`.  (see https://github.com/fossology/fossology/blob/4ed3875bcc096bc093a921613772597a06be76c8/utils/fo-installdeps#L261)